src/routes/ViewAllLogs/ViewAllLogsFilters.js

```diff
@@ -78,6 +78,10 @@ const ViewAllLogsFilters = ({
     label: `${user.firstName} ${user.lastName}`,
   })), 'value');
 
+  const onFilter = (value, dataOptions) => {
+    return dataOptions.filter(o => new RegExp(value, 'i').test(o.label));
```

Comment:
Could you please make more appropriate naming for variable `o`? (e.g. `dataOption` or `option`)